### PR TITLE
Add Wwise Project with Background Music

### PR DESCRIPTION
**Pull request by monikamnc**
🎵 With this PR you will see the new folder for the Wwise project in the root folder. 
🎵 You will find on the Wwise Project that there are some audio files from plugins at this point I am testing with them so I will leave there, when I approach to the final version for sure I will erase it.
🎵 Also, I searched for the suggested by the community files that I should ignore and also I added them to the gitignore file.

🎵 If you want to listen to the Background Music you should uncomment the line 144 on Module Audio. 
🎵 I leave it commented to avoid the programmers hate me and go crazy listening to the music each time are going to compile and test the Engine.

🎵 Thank you for your time ^-^

### CODE REVIEW

**Review by baransrc — CHANGES_REQUESTED** [review]
Thanks for the great work, I've also tested uncommenting the line that fires up the background music and it's great as well!! Just some really small changes are needed to maintain a good quality on what's being merged into our default branch 💯

File: Source/src/modules/ModuleAudio.cpp [hunk 1]
```diff
@@ -1,6 +1,11 @@
 #include "core/hepch.h"
 #include "ModuleAudio.h"
 
+// Bank file names
+#define BANKNAME_INIT L"Init.bnk"
+#define BANKNAME_TEST L"TestBGMusic.bnk"
+#include "..\..\WwiseAkitaInteractive\GeneratedSoundBanks\Wwise_IDs.h" // IDs generated by Wwise
```
[the remark this answers is not in the document]
Comment (baransrc) [reply Source/src/modules/ModuleAudio.cpp:7]: Btw, the illegal characters thing sonarr says is true, `\` without trailing escape like `\\` may make the next character(s) be interpreted as other special characters such as `\n`, and probably by our luck, our compiler tolerates that.

File: Source/src/modules/ModuleAudio.cpp [hunk 2]
```diff
@@ -1,6 +1,11 @@
 #include "core/hepch.h"
 #include "ModuleAudio.h"
 
+// Bank file names
+#define BANKNAME_INIT L"Init.bnk"
+#define BANKNAME_TEST L"TestBGMusic.bnk"
```
[the remark this answers is not in the document]
Comment (monikamnc) [reply Source/src/modules/ModuleAudio.cpp:6]: Thank you for the review and sorry, I followed the tutorials and didn't realize that.
I changed it, take a look if its the way you had in mind or not.
(●ᴥ●)

File: Source/src/modules/ModuleAudio.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (monikamnc) [reply Source/src/modules/ModuleAudio.cpp]: Done, thank you 😺

**Review by baransrc — APPROVED** [review]

**Review by VBoschMatas — APPROVED** [review]

**Pull request merged**